config/default/manager/manager.yaml

```diff
@@ -107,26 +99,68 @@ spec:
         - containerPort: 8000
           name: webhook-server
           protocol: TCP
-        readinessProbe:
-          httpGet:
-            path: /readyz
-            port: webhook-server
-            scheme: HTTPS
-            httpHeaders:
-            - name: Content-Type
-              value: application/json
-          initialDelaySeconds: 10
-          periodSeconds: 10
         volumeMounts:
           - name: tls-key-pair
             readOnly: true
-            mountPath: /etc/webhook/certs
+            mountPath: /tmp/k8s-webhook-server/serving-certs/
       terminationGracePeriodSeconds: 5
       volumes:
         - name: tls-key-pair
           secret:
-            secretName: service
-
+            secretName: kubemacpool-service
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cert-manager
+  namespace: system
+  labels:
+    control-plane: cert-manager
+    controller-tools.k8s.io: "1.0"
+spec:
+  strategy:
+    type: Recreate
+  replicas: 1
+  selector:
+    matchLabels:
+      control-plane: cert-manager
+      controller-tools.k8s.io: "1.0"
+  template:
+    metadata:
+      labels:
+        app: kubemacpool
+        control-plane: cert-manager
+        controller-tools.k8s.io: "1.0"
+    spec:
+      restartPolicy: Always
+      containers:
+      - command:
+        - /manager
+        args:
+          - "--v=production"
+        image: quay.io/kubevirt/kubemacpool:latest
+        imagePullPolicy: Always
+        name: manager
+        env:
+          - name: RUN_CERT_MANAGER
+            value: ""
+          - name: POD_NAMESPACE
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.namespace
+          - name: POD_NAME
```

Comment:
will this have the same POD_NAME as KMP manager?

Reply:
no, the manifests has different names.